apps/archive/archive.py

```diff
@@ -439,6 +439,12 @@ def on_update(self, updates, original):
             publish_from_personal = flask.request.args.get("publish_from_personal") if flask.request else False
             update_state(original, updates, publish_from_personal)
 
+        # set description for custom embed field
+        for key, value in updates.get("extra", {}).items():
+            if type(value) == dict and "embed" in value:
+                value.setdefault("description", "")
+                break
```

Comment:
not sure about this break, if there are more embed field it would only handle 1, why?

Reply:
My bad, forget to remove this.